### PR TITLE
Library App

### PR DESCRIPTION
**Pull request by Fanger53**
## Library
- In this project, we want to make a library app.
-  we use  HTML and Javascript

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### INVALID 🛑 

Hello, @Fanger53 and @Nexch very nice work submitting your project for a review. However, for your project to be reviewed, it has to follow the proper git-flow. Kindly check below ⬇️ 

- [ ] Kindly open a pull request from the feature branch to the development branch. This will ensure the proper use of git-flow.  You have opened a pull request from a feature branch to the main branch which is not the proper way. Kindly fix this. ⬇️ 

git_flow_requirement

But no worries just correct the point above and submit your project for another review.

### if you have any question kindly reach me via slack @peterrobert

### HAPPY CODING 💻

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've done pretty well here with a nice UI and following as much as possible the project requirements. 👌**

However, let's make a couple of changes.

- [ ] Kindly make the development branch the default branch, following the prescribed GitHub workflow, keeping the project professional.

- Please refer to the comment on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: scripts/book.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    this.read = read;
+    this.id = title.slice(0, 3).toUpperCase() + nPages;
+  }
+}
+
+function change(e) {
+  const elem = document.getElementById(e.target.id);
+  if (elem.innerHTML === 'Read') elem.innerHTML = 'Unread';
+  else elem.innerHTML = 'Read';
+}
```
Comment (mikenath223) [inline scripts/book.js:16]: - [ ] Please convert this to an es6 arrow function syntax. It's important to keep up to date on present trends.

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hi team :hand: 
Great job implementing the requested changes.
Good luck with the next project

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**